### ISSUE TITLE
Digital accessibility analysis for Marva Blue Core UI

### ISSUE THREAD

**Issue by kallimathios** [blue-core-lod/marva_editor#23]
An accessibility review should be conducted for the Blue Core Marva editor, as was done for Sinopia Classic. A sister ticket for Sinopia: https://github.com/blue-core-lod/sinopia_editor/issues/50
